internal/command/ping_pong/ping_pong_command.go

```diff
@@ -0,0 +1,18 @@
+package ping_pong
+
+import (
+	"haracho-go/internal/client"
+	"haracho-go/internal/service"
+)
+
+func Init() {
+	help := client.HelpContext{
+		Name:            "!ping",
+		Description:     "PingPong",
+		ArgsDescription: nil,
+	}
+
+	service.GetCommandCollection().AddCommand(&help, "!ping", func(arg *client.Arg, ctx client.CommandContext) {
+		ctx.SendMessage("pong!")
+	})
+}
```

Comment:
```suggestion
func Build(collection *client.CommandCollection) {
	help := client.HelpContext{
		Name:            "!ping",
		Description:     "PingPong",
		ArgsDescription: nil,
	}

	collection.AddCommand(&help, "!ping", func(arg *client.Arg, ctx client.CommandContext) {
		ctx.SendMessage("pong!")
	})
}
```
`CommandCollection` を受け取って書き加えるだけにするべき